configure.ac

```diff
@@ -1093,13 +1093,24 @@ AC_MSG_NOTICE([checking required package dependencies])
 # Check for headers
 #
 
+
+OLD_CFLAGS="${CFLAGS}"
+CFLAGS="${CFLAGS} -Wno-error=address"
 AC_HEADER_STDBOOL
+CFLAGS="${OLD_CFLAGS}"
 AC_HEADER_STDC
 
 AC_CHECK_HEADERS([stdint.h])
 AC_CHECK_HEADERS([string.h])
 
 #
+# Missing Functions
+#
+AC_CHECK_FUNC([strlcpy], [AC_DEFINE([HAVE_STRLCPY], [1], [Define if strlcpy exists.])])
```

Comment:
All you need is `AC_CHECK_FUNC([blahblah])`. It will automatically add `HAVE_BLAHBLAH` to the configure file if appropriate.